src/main/jbake/content/blog/2016/10/24/hawkfx-alerts.adoc

```diff
@@ -0,0 +1,84 @@
+= Define Alert Triggers via HawkFX
+Heiko W. Rupp
+2016-10-24
+:jbake-type: post
+:jbake-status: published
+:jbake-tags: blog, ruby, jruby, javafx, alerts
+
+HawkFX is an explorer for Hawkular. It is a pet project of mine and https://github.com/pilhuhn/hawkfx[hosted on
+Github]. It now got the ability to define alert triggers.
+
+Hawkular-services and now also Hawkular-Metrics standalone have the ability to fire alerts when certain conditions
+are met. To define those conditions, one creates a _trigger_ via
+the http://www.hawkular.org/docs/rest/rest-alerts.html[Hawkular-Alerts api].
```

Comment:
Only pointing to the rest api might not be enough.
I would also add this link http://www.hawkular.org/community/docs/developer-guide/alerts.html.
To understand better what a trigger, condition, event and alert is.